br/use-br-command-line-tool.md

```diff
@@ -244,6 +244,16 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 In the above example, for the incremental backup data, BR records the data changes and the DDL operations during `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR first restores DDL operations and then the data.
 
+### Point in Time recovery (experimental feature)
+
+```shell
+br restore cdclog --storage local:///data/cdclog --start-ts $START_TS --end-ts $END_TS
```

Comment:
Since this is the user document, could you explain a bit about this command? What this command is about? Is it how they can use the Point in Time recovery feature?